src/scripts.js

```diff
@@ -27,20 +23,25 @@ const bigModalCost = document.querySelector('#bigModalCost');
 const bigModalHeart = document.querySelector('#bigModalHeart');
 const bigModalPlus = document.querySelector('#bigModalPlus');
 
+let cookbook;
+let user;
+
+Promise.all([recipeCalls, ingredientCalls, userCalls])
+  .then(data => {
+    const recipes = data[0].recipeData.reduce((sum, recipe) => {
+      sum.push(new Recipe(recipe));
+      return sum;
+    }, []);
+    cookbook = new Cookbook(data[1].ingredientsData, recipes);
+    user = new User(data[2].usersData[getRandomIndex(data[2].usersData)]);
+    startSite();
+  }).catch('this is an error!!!');
```

Comment:
We'll need to check in with others in the cohort about what the catch method should contain.